themes/scss/map/map.scss

```diff
@@ -0,0 +1,5 @@
+.CDB-Map-wrapper {
+  position: relative;
+  height: 100%;
```

Comment:
This is how the DOM looks like after the latest changes:

![screen shot 2017-01-12 at 12 01 13](https://cloud.githubusercontent.com/assets/390398/21887102/d9c633d4-d8be-11e6-9dfa-2facda3d5917.png)